notebooks/individual-dashboards/fairness-dashboard/fairness-dashboard-loan-allocation.ipynb

```diff
@@ -26,8 +26,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
```

Comment:
Could you instead add a comment asking users to uncomment the two lines before running the notebook? Not sure how our test infrastructure responds if these packages are installed from the notebook. And wouldn't users need to restart the kernel when these packages are installed from the notebook?